astroplan/tests/test_core.py

```diff
@@ -155,3 +157,326 @@ def test_FixedTarget_ra_dec():
+    pyephem_next_sunset = datetime.datetime(2000, 1, 1, 18, 3, 23, 676686)
+    pyephem_prev_sunrise = datetime.datetime(2000, 1, 1, 6, 3, 10, 720052)
+    pyephem_prev_sunset = datetime.datetime(1999, 12, 31, 18, 2, 55, 100786)
+
+    # Is there an equivalent to assert_allclose() for datetimes?
+    #assert_allclose(pyephem_next_sunrise, pyephem_next_sunset,
+    #                atol=datetime.timedelta(minutes=10))
+
+    # Typical difference in this example between PyEphem and astroplan is <2 min
+    threshold_minutes = 5
+    assert (abs(pyephem_next_sunrise - astroplan_next_sunrise) <
+            datetime.timedelta(minutes=threshold_minutes))
+    assert (abs(pyephem_next_sunset - astroplan_next_sunset) <
+            datetime.timedelta(minutes=threshold_minutes))
+    assert (abs(pyephem_prev_sunrise - astroplan_prev_sunrise) <
+            datetime.timedelta(minutes=threshold_minutes))
+    assert (abs(pyephem_prev_sunset - astroplan_prev_sunset) <
+            datetime.timedelta(minutes=threshold_minutes))
+
+def get_pyephem_sunrise_sunset():
+    '''
+    Calculate next sunrise and sunset with PyEphem for an observer
+    on the equator.
+    '''
+    lat = '00:00:00'
+    lon = '00:00:00'
+    elevation = 0.0 * u.m
+    pressure = 0
+    time = Time('2000-01-01 12:00:00')
+
+    import ephem
+    obs = ephem.Observer()
+    obs.lat = lat
+    obs.lon = lon
+    obs.elevation = elevation
+    obs.date = time.datetime
+    obs.pressure = pressure
+    next_sunrise = obs.next_rising(ephem.Sun(), use_center=True)
+    next_sunset = obs.next_setting(ephem.Sun(), use_center=True)
+    prev_sunrise = obs.previous_rising(ephem.Sun(), use_center=True)
+    prev_sunset = obs.previous_setting(ephem.Sun(), use_center=True)
+
+    return (repr(next_sunrise.datetime()),
+            repr(next_sunset.datetime()),
+            repr(prev_sunrise.datetime()),
+            repr(prev_sunset.datetime()))
+
+def test_vega_rise_set_equator():
+    '''
+    Check that time of rise/set of Vega for an observer on the equator is
+    consistent with PyEphem results (for no atmosphere/pressure=0)
+    '''
+    lat = '00:00:00'
+    lon = '00:00:00'
+    elevation = 0.0 * u.m
+    pressure = 0
+    location = EarthLocation.from_geodetic(lon, lat, elevation)
+    time = Time('2000-01-01 12:00:00')
+    vega_ra, vega_dec = (279.23473479*u.degree, 38.78368896*u.degree)
+    vega = SkyCoord(vega_ra, vega_dec)
+
+    obs = Observer(location=location, pressure=pressure)
+    astroplan_next_rise = obs.calc_rise(time, vega, which='next').datetime
+    astroplan_next_set = obs.calc_set(time, vega, which='next').datetime
+
+    astroplan_prev_rise = obs.calc_rise(time, vega, which='previous').datetime
+    astroplan_prev_set = obs.calc_set(time, vega, which='previous').datetime
+
+    # Run get_pyephem_vega_rise_set() to compute analogous
+    # result from PyEphem:
+    pyephem_next_rise, pyephem_next_set, pyephem_prev_rise, pyephem_prev_set = (
```

Comment:
I did this because I was lazy and that block of datetimes was the copied and pasted result of the `get_pyephem_*()` method. I'll change this.